libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/location/NavigationLocationProvider.kt

```diff
@@ -172,8 +170,9 @@ class NavigationLocationProvider : LocationProvider {
     ): (ValueAnimator.() -> Unit) {
         val evaluator = PuckAnimationEvaluator(keyPoints)
         return {
-            clientOptions?.also { apply(it) }
+            duration = LocationComponentConstants.DEFAULT_INTERVAL_MILLIS
             evaluator.installIn(this)
+            clientOptions?.also { apply(it) }
```

Comment:
I am also changing the order of options override here to allow `interpolator` and `evaluator` overrides via `clientOptions`.